scripts/actions/edit.py

```diff
@@ -74,7 +74,7 @@
 # the browser but the backend will not let it be updated. This is true
 # for e.g. greyed out fields like forwarded to
 newData = OverriddenData(data, changes)
-for field in changes.keys():
+for field in list(changes.keys()):
     if not isValidField(allFields[field], newData):
         del changes[field]
```

Comment:
Better written as something like this:
```suggestion
changes = {
    field: value
    for field, value in changes.items()
    if isValidField(allFields[field], newData)
}
```